test/index.test.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from "chai";
+import { Chart } from "chart.js";
+
+import ChartUtils from "../src/index.ts";
+
+describe("Testing ChartUtils", () => {
+  it("Testing afterDatasetsUpdate", () => {
+
+    const _meta = {
+      data: [10, 9, 11, 13, 11],
+      $stdevFiller: {},
+    };
+
+    const chart = {
+      data: {
+        datasets: [
+          {
+            label: "example 1",
+            data: [10, 9, 11, 13, 11],
+            stdev: [1, 1.1, 3, 9, 0.2],
+            borderColor: "rgb(0, 0, 0)",
+          },
+        ],
+      },
+      getDatasetMeta: (index) => _meta,
+      isDatasetVisible: (index) => true,
+    };
+
+    ChartUtils.afterDatasetsUpdate(chart);
+
+    const meta = chart.getDatasetMeta(0);
+
+    expect(meta).to.have.property("$stdevFiller");
+    expect(meta.$stdevFiller).to.be.ok
```

Comment:
isn't this line covered by the next?

Reply:
actually first checks that is meta have a property and second one checks if meta.@stdevFiller have some value

Reply:
I was referring to 34 and 35, the first is included in the second as far as I know.